test/__tests__/data/campaign_manager.json

```diff
@@ -356,5 +350,304 @@
         "scope": "exception"
       }
     }
+  },
+  {
+    "description": "Track - batch insert Call failure",
+    "input": {
+      "message": {
+        "channel": "sources",
+        "originalTimestamp": "2022-11-17T00:22:02.903+05:30",
+        "properties": {
+          "floodlightConfigurationId": "213123123",
+          "ordinal": "string",
+          "floodlightActivityId": "456543345245",
+          "value": "756",
+          "quantity": "455678",
+          "requestType": "batchinsert"
+        },
+        "type": "track",
+        "event": "event test",
+        "anonymousId": "randomId",
+        "integrations": {
+          "All": true
+        },
+        "name": "ApplicationLoaded",
+        "sentAt": "2019-10-14T11:15:53.296Z"
+      },
+      "metadata": {
+        "secret": {
+          "access_token": "dummyApiToken",
+          "refresh_token": "efgh5678",
+          "developer_token": "ijkl91011"
+        }
+      },
+      "destination": {
+        "Config": {
+          "profileId": "5343234",
+          "treatmentForUnderage": false,
+          "limitAdTracking": false,
+          "childDirectedTreatment": false,
+          "nonPersonalizedAd": false,
+          "rudderAccountId": "2EOknn1JNH7WK1MfNku4fGYKkRK"
+        }
+      }
+    },
+    "output": {
+      "statusCode": 400,
+      "error": "[CAMPAIGN MANAGER (DCM)]: Atleast one of encryptedUserId,encryptedUserIdCandidates, matchId, mobileDeviceId, gclid, dclid, impressionId.",
+      "statTags": {
+        "destination": "blueshift",
+        "stage": "transform",
+        "scope": "exception"
+      }
+    }
+  },
+  {
+    "description": "Track - batch insert Call",
```

Comment:
This PR doesn't mingle with batching things right ?!

Reply:
yes, it is the name of endinpoint ...nothing to do with batching